### ISSUE TITLE
allow setting of WAI-ARIA role of modal in the invocation

### ISSUE THREAD

**Issue by polyester** [plone/mockup#308]
modal popups become more accessible if the invoking HTML can set role="whatever" on it, just like now they can also provide a css class name

```
<div class="modal-dialog" style="margin: 20px; padding: 0px; width: 400px; left: 442px; top: 0.5px;">    <div class="modal-content">     ...    </div>  </div>
```

would then become 

```
<div class="modal-dialog"  role="whatever" style="margin: 20px; padding: 0px; width: 400px; left: 442px; top: 0.5px;">    <div class="modal-content">     ...    </div>  </div>
```

**Note:** we don't want to set a fixed role of "dialog", because ARIA roles are semantic. If it's not something like a dialog, the author may want to mark it as "complementary" or any of the other roles available. That should be left to the html that uses the pattern. And actually making use of the role is the job of the client.

Browsers and screenreaders are fine with empty strings,so role="" as a default would be ok.

**Comment by jensens** [comment]
Closing as outdated. Mockup was completely rewritten for Plone 6 (jQuery/Backbone/RequireJS → ES6/Patternslib). Old pat-modal ARIA role request.
If you think this issue is still relevant in the new implementation, please reopen it. Thank you for the original report!
